_studio/mfx_lib/CMakeLists.txt

```diff
@@ -155,12 +155,18 @@ else()
   endif()
 endif()
 
-if( API_VERSION VERSION_GREATER 1.25 )
+if( ${MFX_ENABLE_MCTF} )
```

Comment:
good catch, but this is a separate change non related to the ASC:). And you also forget this place: https://github.com/Intel-Media-SDK/MediaSDK/blob/ed67603472e33ef03ce92a80f28a55e3e7b78d65/_studio/mfx_lib/genx/CMakeLists.txt#L79

Don't create separate PR for this though...